src/lavinmq/clustering/follower.cr

```diff
@@ -164,11 +167,14 @@ module LavinMQ
         lag_size
       end
 
-      @closed = Channel(Nil).new
+      @closed_change = Channel(Nil).new
```

Comment:
use a better name? `in_sync`, `closed_and_in_sync`, something else? 

```suggestion
      @closed_and_in_sync = Channel(Nil).new
```

Reply:
Yeah, I think `closed_and_in_sync` conveys what it does pretty well 👍